staff/robert-anthony/spotify-app/styles/main.scss

```diff
@@ -0,0 +1,172 @@
+$yellowy: rgb(255, 251, 0);
+$orangey: rgb(255, 123, 0);
+
+@mixin detailsMixin {
+  height: min-content;
+  width: 100%;
+  color: white;
+  font-weight: 300;
+  padding: 2em;
+  padding-bottom: 3em;
+  border: 4px dotted white;
+  list-style-type: none;
+  visibility: hidden;
+  li a {
+    @include link-styling;
+  }
+
+  button {
+    @include button-mix;
+
+  }
+  h1 {
+    @include h1-mix;
+  }
+  ul {
+    list-style-type: none;
+    li {
+      margin: 0;
+      padding: 0;
+    }
+  }
+  span {
+    font-weight: 700;
+    color: $orangey;
+  }
+}
+
+@mixin button-mix {
+  margin: 0;
+  padding: 4px;
+  font-size: 1.2em;
+  background: white;
+  color: #000eb3;
+  -webkit-border-radius: 10%;
+  -moz-border-radius: 10%;
+  border-radius: 10%;
+  cursor: pointer;
+  &:hover {
+    color: $orangey;
+  }
+}
+
+@mixin h1-mix {
+  font-weight: 900;
+  color: $yellowy;
+  letter-spacing: .2em;
+}
+
+@mixin link-styling {
+  text-decoration: none;
+  cursor: pointer;
+  margin: 0;
+  padding: 0;
+  color: inherit;
+  &:hover {
+    color: $orangey;
+  }
+}
+
+* {
+  -webkit-box-sizing: border-box;
+  -moz-box-sizing: border-box;
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  font-family: 'Lato', sans-serif;
+  font-weight: 400;
+}
+
+.mainContainer {
```

Comment:
It's more elegant to use kebab-case in class name
```css
.main-container {
```

Reply:
Either/or